hw/drivers/pwm/include/pwm/pwm.h

```diff
@@ -121,18 +134,43 @@ struct pwm_dev {
  *
  * pin - The pin to be assigned to this pwm channel.
  * inverted - Whether this channel's output polarity is inverted or not.
- * data - A pointer do a driver specific parameter.
+ * n_cycles - The number of cycles for the channel to play. 0 for loop mode.
+ * interrupts - Whether interruptions are to be used or not.
+ * data - Driver specific data.
```

Comment:
Would it make sense to add a comment here that if `interrupts == true` this has to be a point to a `struct pwm_dev_interrupt_cfg` ?

Reply:
never mind - I misread the interface.

Reply:
Eventhough the comment on 'interrupts' covers both possible states, it is misleading, sorry.